### PR TITLE
Emby.Drawing.Skia remove deprecated functions.

### PR DESCRIPTION
**Pull request by EraYaN**
Cleaned up some other declarations along the way.

### CODE REVIEW

File: Emby.Drawing.Skia/SkiaEncoder.cs [hunk 1]
```diff
@@ -548,17 +548,17 @@ public string EncodeImage(string inputPath, DateTime dateModified, string output
...
 
                     // If all we're doing is resizing then we can stop now
                     if (!hasBackgroundColor && !hasForegroundColor && blur == 0 && !hasIndicator)
                     {
                         _fileSystem.CreateDirectory(_fileSystem.GetDirectoryName(outputPath));
                         using (var outputStream = new SKFileWStream(outputPath))
                         {
-                            resizedBitmap.Encode(outputStream, skiaOutputFormat, quality);
+                            SKImageInfo imageInfo = new SKImageInfo(width,height);
+                            var pixmap = new SKPixmap(new SKImageInfo(width, height), resizedBitmap.GetPixels());
```
Comment (Bond-009) [inline Emby.Drawing.Skia/SkiaEncoder.cs]: `SKPixmap` implements `IDisposable`, maybe wrap it in a `using` statement?

Comment (EraYaN) [reply Emby.Drawing.Skia/SkiaEncoder.cs]: It should dispose immediately after the return anyway, but it's probably nicer to do it explicitly.

File: Emby.Drawing.Skia/SkiaEncoder.cs [hunk 2]
```diff
@@ -556,10 +556,11 @@ public string EncodeImage(string inputPath, DateTime dateModified, string output
                         _fileSystem.CreateDirectory(_fileSystem.GetDirectoryName(outputPath));
                         using (var outputStream = new SKFileWStream(outputPath))
                         {
-                            SKImageInfo imageInfo = new SKImageInfo(width,height);
-                            var pixmap = new SKPixmap(new SKImageInfo(width, height), resizedBitmap.GetPixels());
-                            pixmap.Encode(outputStream, skiaOutputFormat, quality);
-                            return outputPath;
+                            using (var pixmap = new SKPixmap(new SKImageInfo(width, height), resizedBitmap.GetPixels()))
```
Comment (Bond-009) [inline Emby.Drawing.Skia/SkiaEncoder.cs]: You don't need another indentation level for 2 using statements, you can do:
```cs
using (var stuff = foo)
using (var morestuff = foo)
{ // Use stuff and morestuff }

Comment (Bond-009) [reply Emby.Drawing.Skia/SkiaEncoder.cs]: Yes please, with Lukes indentation the lines are halfway my ultrawide.

Comment (EraYaN) [reply Emby.Drawing.Skia/SkiaEncoder.cs]: Will postpone the reformatting after #382 is merged, and then auto reformat.

**Comment by EraYaN** [comment]
The indentation is in-line with the current format settings of #382 I believe.

**Comment by JustAMan** [comment]
> The indentation is in-line with the current format settings of #382 I believe.

I was talking about @Bond-009 comments and too many nested levels. I agree that it makes sense to auto-format, though.

**Comment by EraYaN** [comment]
I know but there is a setting for that as well.

```cs
if (true)
   a = 6;
```

vs 
```cs
if (true)
{
    a = 6;
}
```

So those nested `using` clauses will format either way depending. The second one is default and ".NET" style of what ever that means. Doesn't seem to offer seperate settings. It's governed by the `csharp_prefer_braces`. [reference](https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference?view=vs-2017) sadly no deep linking in that page. Ctrl/Cmd+F will have to do. I would argue it has a slightly detrimental effect removing that, but it's preference and I'm on a large screen.

**Pull request merged**